docs/admin/bootstrap-tokens.md

```diff
@@ -156,3 +175,8 @@ is then used to form a whole JWS by inserting it between the 2 dots.  You can
 verify the JWS using the `HS256` scheme (HMAC-SHA256) with the full token (e.g.
 `07401b.f395accd246ae52d`) as the shared secret.  Users _must_ verify that HS256
 is used.
+
+WARNING: Any party with a bootstrapping token can create a valid signature for that
```

Comment:
There's a little section in the `kubeadm` docs that talks about this, not sure if it's worth linking or not: https://kubernetes.io/docs/reference/generated/kubeadm/#security-model

+1 on calling out this risk either way.

Reply:
Didn't know that doc existed, thanks!